src/platform/visionfive2.rs

```diff
@@ -79,8 +79,8 @@ impl Platform for VisionFive2Platform {
     }
 
     fn get_miralis_memory_start_and_size() -> (usize, usize) {
-        let size = FIRMWARE_START_ADDR - MIRALIS_START_ADDR;
-        (MIRALIS_START_ADDR, size)
+        //let size = FIRMWARE_START_ADDR - MIRALIS_START_ADDR;
+        (MIRALIS_START_ADDR, 0x200000) // Temporary workaround, suggestions appreciated
```

Comment:
In the linker script we have a symbol `_start_address` for the start of Miralis, and a symbol `_stack_start` for the start of the stack. The total size of Miralis is `_stack_start - _start_address + PLATFORM_STACK_SIZE * PLATFORM_NB_HARTS`